src/pages/references/authoring/editor/classes/Editor.md

```diff
@@ -119,9 +114,8 @@ This local client will act as having unsaved changes until the upload has finish
 
 | Name | Type | Description |
 | :------ | :------ | :------ |
-| `bitmapData` | [`BitmapImage`](../interfaces/BitmapImage.md) | BitmapImage resource (e.g. returned from loadBitmapImage()). |
-| `options` | `Object` | Additional configuration: |
-| `options.initialSize?` | [`RectangleGeometry`](../interfaces/RectangleGeometry.md) | Size the image is displayed at. Must have the same aspect ratio as bitmapData. Defaults to the        size the image would be created at by a UI drag-drop gesture (typically the image's full size, but scaled down        if needed to stay below an application-defined size cap). |
+| `bitmapData` | [`BitmapImage`](../interfaces/BitmapImage.md) | BitmapImage resource (e.g. returned from `loadBitmapImage()`). |
```

Comment:
We should make sure to get the added "\`"s here updated in the original upstream docs too...

I'm happy to help with that, we just need to make sure to note down any time we change something in the generated docs to make sure we can propagate it back.